pyproject.toml

```diff
@@ -1,28 +1,32 @@
-[tool.poetry]
+[project]
 name = "beng-lod-server"
-version = "1.0"
 description = "Webserver serving LOD for the NISV catalogue items, datasets and GTAA."
-authors = [
-  "Willem Melder <wmelder@beeldengeluid.nl>",
-  "Mari Wigham <mwigham@beeldengeluid.nl>",
-  "Govert Brinkmann <gbrinkmann@beeldengeluid.nl>"
-  ]
-license = "MIT"
+version = "2.1.2"
+requires-python = "~=3.11.0"
 readme = "README.md"
-packages = []
-package-mode = false
+license = "MIT"
+authors = [
+  {name = "Willem Melder", email = "wmelder@beeldengeluid.nl"},
+  {name = "Govert Brinkmann", email = "gbrinkmann@beeldengeluid.nl"},
+]
+dependencies = [
+  "requests (==2.32.4)",
+  "rdflib (==7.1.4)",
+  "validators (==0.35.0)",
+  "pyyaml (==6.0.2)",
+  "Flask-Cors (==6.0.1)",
+  "flask (==3.1.1)",
+  "flask-restx (==1.3.0)",
+  "gunicorn (>=23.0.0)",
```

Comment:
`gunicorn` had `^23.0.0`, would at least add upper bound of 24.0.0 to get something equivalent.